modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -638,9 +639,29 @@ private void _validate(
 				"Invalid output type " + outputType);
 		}
 
-		ObjectValidationRuleEngine objectValidationRuleEngine =
-			_objectValidationRuleEngineRegistry.getObjectValidationRuleEngine(
-				companyId, engine);
+		ObjectValidationRuleEngine objectValidationRuleEngine = null;
+
+		try {
+			objectValidationRuleEngine =
+				_objectValidationRuleEngineRegistry.
+					getObjectValidationRuleEngine(companyId, engine);
+		}
+		catch (PortalException portalException) {
+			if (portalException instanceof
+					ObjectValidationRuleEngineException.NoSuchEngine) {
+
+				if (_log.isDebugEnabled()) {
+					_log.debug(
+						StringBundler.concat(
+							"No object validation rule engine is registered ",
+							"with key ", engine));
+				}
+
+				return;
+			}
+
+			throw portalException;
+		}
```

Comment:
Hey @pedro-oliveira446 use the same approach used for object actions:

1. Create a method that check if the a given object validation rule engine exists ([see](https://github.com/liferay/liferay-portal/blob/80d9c57eea909f96a15df84d4dcfd811329b0d33/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/ObjectActionExecutorRegistryImpl.java#L108-L110))
2. use this method [this way](https://github.com/liferay/liferay-portal/blob/49fa1bb81c1fbfd550233fca965ab3d2f46f1b5a/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java#L522-L532)
3. In the integration test you can use the ExpectedLogMethodTestRule, see [this example](https://github.com/liferay/liferay-portal/blob/545a1284e57888bc99853ecbfd2e124c8d00c308/modules/apps/portal-search-opensearch2/portal-search-opensearch2-impl/src/test/java/com/liferay/portal/search/opensearch2/internal/logging/OpenSearchIndexSearcherLogExceptionsOnlyTest.java#L43-L47)

Thank you!